packages/cli/package.json

```diff
@@ -48,6 +48,7 @@
     "fs-extra": "^7.0.1",
     "json5": "^2.0.1",
     "lodash": "^4.17.11",
+    "loopback-datasource-juggler": "^4.5.3",
```

Comment:
IIUC, `loopback-datasource-juggler` is used only by the test suite. If that's true, then please move it from `dependencies` to `devDependencies`.

Reply:
WOops, fixed